File: BackEnd/TasksTracker/app/models/initializers/prioritiesInitializer.js
const Priorities = require('../Priority');

const priorities = [
  {
    value: 1,
    description: 'Very low'
  },
  {
    value: 2,
    description: 'Low'
  },
  {
    value: 3,
    description: 'Normal'
  },
  {
    value: 4,
    description: 'High'
  },
  {
    value: 5,
    description: 'Very high'
  }
]


module.exports = initializePriorities = async () => {
  const result = await Priorities.insertMany(priorities);
  console.log('inserted priorities:', result);
}

File: BackEnd/TasksTrackerNet/TasksTracker/TasksTracker/Database/Initializers/PrioritiesInitializer.cs
using Microsoft.EntityFrameworkCore;
using System.Collections.Generic;
using System.Linq;
using TasksTracker.Database.Context;
using TasksTracker.Interfaces;
using TasksTracker.Models.DatabaseModels;

namespace TasksTracker.Database.Initializers
{
    public class PrioritiesInitializer : IDatabaseInitializer
    {
        public void Initialize(DatabaseContext context)
        {
            context.Database.Migrate();
            if (context.Priorities.Any()) { return; }
            var priorities = GetPriorities();
            context.Priorities.AddRange(priorities);
            context.SaveChanges();
        }

        public List<Priority> GetPriorities()
        {
            return new List<Priority>
            {
              new Priority
              {
                Value = 1,
                Description = "Very low"
              },
              new Priority
              {
                Value=2,
                Description= "Low"
              },
              new Priority
              {
                Value=3,
                Description= "Normal"
              },
              new Priority
              {
                Value=4,
                Description= "High"
              },
              new Priority
              {
                Value= 5,
                Description= "Very high"
              }
            };
        }

    }
}
